sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/Document.java

```diff
@@ -21,6 +23,16 @@
  */
 public class Document extends Resource {
 
+    /**
+     * Constructor.
+     *
+     * @param objectNode the {@link ObjectNode} that represent the
+     * {@link JsonSerializable}
+     */
+    public Document(ObjectNode objectNode) {
```

Comment:
Same here.

Reply:
all of the new public constructors are in the implementation package not in the public surface area.